examples/angular2/services/store.ts

```diff
@@ -1,9 +1,5 @@
-/// <reference path="../typings/node-uuid/node-uuid.d.ts" />
-/// <reference path="../typings/storejs/storejs.d.ts" />
-/// <reference path="../typings/angular2/angular2.d.ts" />
-
-import {Injectable} from 'angular2/angular2';
 import * as uuid from 'node-uuid';
+import 'store.js';
```

Comment:
> My understanding is that the type definitions for store.js define a global variable rather than a module:

Sounds like the type definition is wrong or outdated.

https://github.com/marcuswestin/store.js/blob/b115845a60bb0c3922ec1ddfd1d35762cf0fec30/store.js#L12

> To be honest, I'm not sure why this example uses store.js rather than localStorage directly? 

Yeah me neither, I think we should just move to vanilla localStorage, but can be in your followup PR.